<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMeasurementsDialog</class>
 <widget class="QDialog" name="NewMeasurementsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="last_measurements_group">
     <property name="title">
      <string>Measurements</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Weight:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="weight_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Shoulder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="shoulder_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Chest:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="chest_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Arm:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="arm_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Belly:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="belly_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Hip:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hip_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Leg:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leg_line_edit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,10">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Taken Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="taken_date_edit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2024</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="save_push_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
